dds_cli/account_manager.py

```diff
@@ -335,6 +335,10 @@ def fix_project_access(self, email, project):
                 json=json,
                 timeout=dds_cli.DDSEndpoint.TIMEOUT,
             )
+            if response.ok:
+                LOG.info(
+                    f"[green] :white_check_mark: Project access fixed for user '{email}'.[/green]"
+                )
```

Comment:
Seems like the issue is only that the `msg` on line 374 is not printed out, so I suggest that we remove this and instead we add `dds_cli.utils.console.print(msg)` on line 381, outside of the if / else
```suggestion
```